internal/cmd/beta/server/backup/create/create.go

```diff
@@ -0,0 +1,171 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/goccy/go-yaml"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	cliErr "github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/serverbackup/client"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/serverbackup"
+)
+
+const (
+	backupNameFlag            = "name"
+	backupRetentionPeriodFlag = "retention-period"
+	backupVolumeIdsFlag       = "volume-ids"
+	serverIdFlag              = "server-id"
+
+	defaultRetentionPeriod = 14
+	defaultVolumeIds       = ""
```

Comment:
If this flag is changed for a slice flag, then probably we don't need this default to be explicit